styles.css

```diff
@@ -334,3 +369,105 @@ p {
 .skills .list .item {
   font-size: 2.2rem;
 }
+
+/* Footer */
+.footer {
+  position: relative;
+  top: 100vh;
+}
+
+/* contact form */
+
+.contact {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 7rem 0;
+  gap: 2.4rem;
+}
+
+.footer .contact h2 {
+  color: var(--dark-font-color);
+  flex: 1;
+  font-size: 3.2rem;
+  line-height: 4.5rem;
+  padding-bottom: 1.2rem;
+  text-align: center;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 2.4rem;
+  width: 100%;
+}
+
+.form .text {
+  border-bottom: 1px solid var(--hero-icon-color);
+  font-size: 1.5rem;
+  padding: 1.5rem;
+  width: 100%;
+}
+
+.form .text::placeholder {
+  color: var(--form-text-color);
+  font-weight: 100;
+  opacity: 0.8;
+}
+
+.firstName,
+.lastName {
+  display: none;
+}
+
+.form .area {
+  background-color: var(--area-bg-color);
+  font-size: 1.5rem;
+  font-weight: 400;
+  height: 12rem;
+  padding: 1.2rem;
+  width: 100%;
+}
```

Comment:
- [ ] To correctly match the Figma design and as it is required for the project, kindly ensure that you use the approppriate `font-family` in your `text-area` field as now the used one is different to the one from the template design which is `Inter`.

Reply:
Hi @tresorsawasawa. Thank you for pointing this out, I completely missed it. 🤪
I made the changes.🙏